src/client.cpp

```diff
@@ -704,6 +706,17 @@ void CClient::OnControllerInFaderIsMute ( int iChannelIdx, bool bIsMute )
     emit ControllerInFaderIsMute ( iChannelIdx, bIsMute );
 }
 
+void CClient::OnControllerInMuteMyself ( bool bMute )
+{
+    // in case of a headless client the buttons are not displayed so we need
+    // to send the controller information directly to the server
+#ifdef HEADLESS
+    // FIXME: no idea what to do here.
```

Comment:
Basically, follow the path.

What the `emit` does is handled somewhere - that method ends up back in the client code and should be what's called here.

There should really be _no_ `client.cpp` code that's dependent upon the state of `#ifdef HEADLESS` - the client should work the same regardless.